test-data/unit/check-expressions.test

```diff
@@ -604,7 +604,7 @@ class X:
 class Y:
     def __lt__(self, o: 'Y') -> A: pass
     def __gt__(self, o: 'Y') -> A: pass
-    def __eq__(self, o: 'Y') -> B: pass
+    def __eq__(self, o: 'Y') -> B: pass  # type: ignore
```

Comment:
Why is this needed?

Reply:
I updated the fixture (to be more like in typeshed) and this now causes incompatible with supertype.